src/components/orgunits/OrgUnitSelect.js

```diff
@@ -3,16 +3,16 @@ import { useDataQuery } from '@dhis2/app-runtime'
 import { CenteredContent, CircularLoader, Help } from '@dhis2/ui'
 import cx from 'classnames'
 import PropTypes from 'prop-types'
-import React, { useCallback, useEffect } from 'react'
+import React, { useCallback } from 'react'
 import { useSelector, useDispatch } from 'react-redux'
 import { setOrgUnits } from '../../actions/layerEdit.js'
-import { DEFAULT_ORG_UNIT_LEVEL } from '../../constants/layers.js'
 import { translateOrgUnitLevels } from '../../util/orgUnits.js'
 import AssociatedGeometrySelect from './AssociatedGeometrySelect.js'
 import OrgUnitSelectMode from './OrgUnitSelectMode.js'
 import styles from './styles/OrgUnitSelect.module.css'
 
-// Fetches the root org units associated with the current user with fallback to data capture org units
+// Fetches org unit levels and the root org units associated with the current user
+// with fallback to data capture org units
```

Comment:
Is this query still needed? Looks exactly the same as the one in the Provider, so is it possible to use the `useOrgUnits` hook instead?

Reply:
My bad - fixed!